nix-diff.cabal

```diff
@@ -30,6 +30,7 @@ executable nix-diff
                      , text                 >= 1.2      && < 1.3
                      , unix                                < 2.8
                      , vector               >= 0.12     && < 0.13
+                     , process
```

Comment:
What would be the reasonable version constraints ? Just `< 1.7` ?

Reply:
Yeah, `< 1.7` is what I recommend

Reply:
Following the same logic, I added `< 1.5` for filepath.